src/common/options.cc

```diff
@@ -327,6 +327,9 @@ std::vector<Option> get_global_options() {
     .set_daemon_default(true)
     .set_description("send critical error log lines to stderr"),
 
+    Option("log_stderr_prefix", Option::TYPE_STR, Option::LEVEL_ADVANCED)
+    .set_description("String to prefix log messages with when sent to stderr"),
```

Comment:
mind setting a long description containing a brief example about when this would be useful?

Reply:
see https://github.com/ceph/ceph/pull/19869

Reply:
thx